### ISSUE TITLE
Explore the Neuralforecast in the 16-homesale-forecasting.ipynb

### ISSUE THREAD

**Issue by xzdandy** [georgia-tech-db/evadb#1236]
### Search before asking

- [X] I have searched the EvaDB [issues](https://github.com/georgia-tech-db/eva/issues) and found no similar feature requests.

### Description

After #1112 is merged into staging, we'd like to explore the Neuralforecast in the homesale forecasting notebook to see if Neuralforecast addresses the limitation we have with statsforecast. 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!
